app/views/application/_button_edit.html.slim

```diff
@@ -0,0 +1,3 @@
+.row
+  .col-xs-12
+    = link_to t_action(:edit), { controller: controller_name, action: :edit, id: params[:id] }, { class: 'btn btn-default' }
```

Comment:
can't follow here. 

old version:
`link_to t_action(:edit), edit_client_path(@client), class: 'btn btn-default'`

with this new way - how does it know to go on which path? 
and the `@client` is here then directed with the` id: params[:id]` ?

need explanations here for better understanding the refactoring